Clarification of Customer Comment # 20231012 (#6373)
opensearch-project/documentation-website: _security/multi-tenancy/tenant-index.md

- **Private** -- This tenant is exclusive to each user and can't be shared. It does not allow you to access routes or index patterns created by the user's global tenant.
- **Custom** -- Administrators can create custom tenants and assign them to specific roles. Once created, these tenants can then provide spaces for specific groups of users.

> The global tenant in OpenSearch Dashboards doesn't synchronize its content with private tenants. When you make modifications within your global tenant, these changes are exclusive to the global tenant. They aren't automatically mirrored or replicated in the private tenant. Some example changes include the following:

Reviewer: ```suggestion
The global tenant in OpenSearch Dashboards doesn't synchronize its content with private tenants. When you make modifications within your global tenant, these changes are exclusive to the global tenant. They aren't automatically mirrored or replicated in the private tenant. Some example changes that are exclusive to the global tenant include the following:
```

Author: Hi @Naarcha-AWS, the operations listed:
Create visualizations
Create index patterns
Are not exclusive to only the global tenant so I think the new wording will be misleading.

Reviewer: @leanneeliatra: I removed that part of the sentence. Let me know if this looks okay.

Author: I'm not sure how to 'mark the requested changes are addressed', but once that is complete, this all looks good to be merged now. Thanks!

> The global tenant in OpenSearch Dashboards doesn't synchronize its content with private tenants. When you make modifications within your global tenant, these changes are exclusive to the global tenant. They aren't automatically mirrored or replicated in the private tenant. Some example changes including the following:

Reviewer: ```suggestion
The global tenant in OpenSearch Dashboards doesn't synchronize its content with private tenants. When you make modifications within your global tenant, these changes are exclusive to the global tenant. They aren't automatically mirrored or replicated in the private tenant. Some example changes to both private and global tenants include the following:
```

> The global tenant in OpenSearch Dashboards doesn't synchronize its content with private tenants. When you make modifications within your global tenant, these changes are exclusive to the global tenant. They aren't automatically mirrored or replicated in the private tenant. Some example changes to both private and global tenants include the following:

Reviewer: ```suggestion
The global tenant in OpenSearch Dashboards doesn't synchronize its content with private tenants. When you make modifications inside your global tenant, these changes are exclusive to the global tenant. They aren't automatically mirrored or replicated in the private tenant. Some example changes to both private and global tenants include the following:
```
